src/lib/functions/netlify-function.js

```diff
@@ -41,7 +41,9 @@ class NetlifyFunction {
     // Determines whether this is a background function based on the function
     // name.
     this.isBackground = name.endsWith(BACKGROUND_SUFFIX)
-    this.schedule = null
+
+    const scheduledFunc = config && config.functions && config.functions[name]
```

Comment:
I believe `config` is always defined.

```suggestion
    const scheduledFunc = config.functions && config.functions[name]
```

Reply:
sand: You don't know that this is a scheduled function, so I think `scheduledFunc` is not a great name for this variable. Perhaps something like `functionConfig` would be more accurate?

Reply:
> I believe `config` is always defined.

I added the additional check because a few tests were failing.